stock_quant_package_fast_move/wizard/stock_quant_package_fast_move_wizard.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, exceptions, fields, models
+
+
+class StockQuantPackageFastMoveWizard(models.TransientModel):
+    _name = "stock.quant.package.fast.move.wizard"
+    _description = "Stock Quant Package Fast Move Wizard"
+
+    package_dest_id = fields.Many2one(
+        "stock.quant.package",
+        "Destination Package",
+        domain="[('location_id', '=', location_dest_id)]",
+    )
+    location_dest_id = fields.Many2one(
+        "stock.location", "Destination Location", required=True
+    )
+    put_in_new_package = fields.Boolean("Put in New Package")
+
+    def action_move(self):
+        """
+        Execute the package movement based on the specified parameters.
+        """
+        package_ids = self._context.get("active_ids")
+        packages = self.env["stock.quant.package"].browse(package_ids)
+        if any(package.location_id != packages[0].location_id for package in packages):
+            raise exceptions.UserError(
+                _("All packages must have the same source location.")
+            )
+
+        if self.put_in_new_package:
+            destination_package_id = packages.create_new_package(self.location_dest_id)
```

Comment:
You pass 1 argument to the "create_new_package function", but it takes no arguments

Reply:
Fixed! Thanks for the catch!